applications/StructuralMechanicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -30,6 +30,9 @@
 #include "custom_processes/impose_rigid_movement_process.h"
 #include "custom_processes/impose_z_strain_process.h"
 #include "custom_processes/distribute_load_on_surface_process.h"
+#include "custom_processes/set_cartesian_local_axes_process.h"
+#include "custom_processes/set_cylindrical_local_axes_process.h"
+#include "custom_processes/set_spherical_local_axes_process.h"
```

Comment:
This adds lots of files for one different function each!
can you please put it into one utility (and hence follow the style of the already existing `ProjectToSurfaceUtility`)?

esp since you are also adding three python processes I don't see the necessity to add also the same in C++

Reply:
I did it like you say but @RiccardoRossi told me that was better to separate them :D

Reply:
@RiccardoRossi can you comment please?
I am not sure if adding 6 files for 3 functions makes sense

sorry for delaying @AlejandroCornejo  :/